package.json

```
@@ -6,20 +6,29 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "web": "expo start --web"
+    "web": "expo start --web",
+    "format:fix": "prettier --write . && eslint --fix ."
```

```suggestion
    "format": "prettier --write .",
    "lint": "eslint --fix ."
```

Separated linting and formatting